src/batch/src/executor/iceberg_scan.rs

```diff
@@ -334,149 +367,3 @@ impl PositionDeleteFilter {
         self.position_delete_file_path_pos_map.remove(file_path);
     }
 }
-
-struct EqualityDeleteFilter {
```

Comment:
Are the following logics shifted elsewhere? Or are they invalidated for some reason?

Reply:
After this pr, this logic is no longer valid and we have implemented the same functionality using hash join exec